<launch>
  <node name="keyboard_teleop" type="keyboard_teleop.py" pkg="mf_robot_simulator"
    output="screen">
    <remap from="command" to="/simu/robot_simulator/input" />
    <remap from="cart_command" to="/simu/robot_simulator/cart_input" />

    <param name="init_speed" value="500.0" />
    <param name="init_cmd_vx" value="0.3" />
    <param name="init_cmd_vy" value="0.5" />
    <param name="init_cmd_vz" value="0.5" />
    <param name="init_speed" value="500.0" />
    <param name="inc_delta" value="1.0" />
    <param name="inc_P" value="0.002" />
  </node>
</launch>
